src/services/apiService.js

```diff
@@ -89,6 +89,15 @@ export default class ApiService {
     return this.makeRequestOnPosts('POST', postType, fields);
   }
 
+    /**
+   * Fetch contacts list via API
+   * @param {string} postType
+   * @returns {Promise<any>}
+   */
+  async fetchContactsList(postType) {
```

Comment:
This would probably be better named `fetchPostsList` since the list could contain any number of post types like contacts or groups

Reply:
done